src/datasets/other/serp_events/reference.md

```diff
@@ -430,73 +432,108 @@ Number of SERP impressions with ads loaded and an ad blocker in use:
+| `ad_components`                              | Array with 1 entry for each ad component which had either an impression or engagement, `[]` if none.                                                                                                                                                         |
+| `ad_components.component`                    | SERP display component containing ad links [e.g. `ad_link`, `ad_carousel`].                                                                                                                                                                                  |
+| `ad_components.num_loaded`                   | Number of ads loaded in the component. They may or may not be visible on the page.                                                                                                                                                                           |
+| `ad_components.num_visible`                  | Number of ads visible to the user in the component.                                                                                                                                                                                                          |
+| `ad_components.num_blocked`                  | Number of ads blocked by an ad blocker in the component.                                                                                                                                                                                                     |
+| `ad_components.num_notshowing`               | Number of ads in the component which are loaded but not visible, and not blocked by an ad blocker. For example, ads in the carousel that will be shown on clicking the "Expand" button.                                                                      |
+| `ad_components.num_clicks`                   | Number of clicks on ad links in the component.                                                                                                                                                                                                               |
+| `ad_components.num_other_engagements`        | Number of engagements in the component which are not ad clicks. E.g. clicking "Expand" for the carousel.                                                                                                                                                     |
+| `ad_components.blocker_inferred`             | Can we infer an ad blocker is in use in the component? `true` if all loaded ads are blocked, `false` otherwise. Note that the same SERP impression can have `blocker_inferred = true` for some ad components and `false` for others.                         |
```

Comment:
Turn first sentence into a statement rather than a question, could even just remove it